Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeHostIOS.mm

```diff
@@ -152,7 +152,7 @@
 #endif
 
         if (!modelHandledOutOfProcess && m_drawingArea->page() && m_drawingArea->page()->preferences().modelElementEnabled()) {
-            if (auto* model = std::get_if<Ref<Model>>(&properties.additionalData)) {
+            if (std::get_if<Ref<Model>>(&properties.additionalData)) {
 #if ENABLE(SEPARATED_MODEL)
                 return makeWithView(adoptNS([[WKSeparatedModelView alloc] initWithModel:*model]));
```

Comment:
I think we need to keep the `model` variable? It's used on line 157.

Reply:
Yeah, this change should be #else UNUSED_PARAM(model); return nullptr;

Reply:
Oh of course, thanks for catching!